base::FileTracing::Provider has virtual functions but no virtual dtor.
This trips -Wnon-virtual-dtor warning on CrOS side.

diff --git a/base/files/file_tracing.h b/base/files/file_tracing.h
index 149bd78..aca1ff7 100644
--- a/base/files/file_tracing.h
+++ b/base/files/file_tracing.h
@@ -27,6 +27,8 @@ class BASE_EXPORT FileTracing {
  public:
   class Provider {
    public:
+    virtual ~Provider() = default;
+
     // Whether the file tracing category is currently enabled.
     virtual bool FileTracingCategoryIsEnabled() const = 0;
 
